pkg/tools/tools.go

```diff
@@ -11,5 +11,5 @@ For additional documentation on the topic:
 package tools
 
 import (
-	_ "github.com/go-bindata/go-bindata/go-bindata/"
+	_ "github.com/go-bindata/go-bindata/go-bindata"
```

Comment:
kept getting the error:

```
❯  go mod tidy
go: github.com/kolide/launcher/pkg/tools imports
        github.com/go-bindata/go-bindata/go-bindata/: malformed import path "github.com/go-bindata/go-bindata/go-bindata/": trailing slash
```

until I updated this, which is the opposite of

https://github.com/kolide/launcher/issues/1295

🤷  .... future me will figure this out

Reply:
.... aannnnnd it's gone